.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+	"recommendations": [
+		"llvm-vs-code-extensions.vscode-clangd",
+		"vadimcn.vscode-lldb",
+		"rust-lang.rust-analyzer",
+		"tamasfe.even-better-toml",
+		"serayuzgur.crates",
+		"jac3km4.redscript-ide-vscode",
+		"jackhumbert.redscript-syntax-highlighting",
+		"skellock.just"
```

Comment:
these 5 are pretty optional, I wouldn't want to spam too many recommendations for things that aren't necessarily useful, so I'd keep only the first three